dask_kubernetes/operator/core.py

```diff
@@ -0,0 +1,88 @@
+# import asyncio
+# import subprocess
+
+from distributed.deploy import Cluster
+from distributed.core import rpc
+
+# import kopf
+import kubernetes
+
+from dask_kubernetes.utils import (
+    #     namespace_default,
+    get_external_address_for_scheduler_service,
+    #     check_dependency,
+)
+
+from daskcluster import (
+    build_cluster_spec,
+)  # , wait_for_scheduler, build_worker_group_spec
+
+
+class KubeCluster2(Cluster):
+    """Launch a Dask Cluster on Kubernetes using the Operator"""
+
+    def __init__(
+        self,
+        name,
+        namespace="default",
+        image="daskdev/dask:latest",
+        replicas=3,
+        resources={},
+        env={},
+        loop=None,
+        asynchronous=False,
+        **kwargs
+    ):
+        self.name = name
+        self.namespace = namespace
+        self.core_api = None
+        self.custom_api = None
+        self.image = image
+        self.replicas = replicas
+        self.resources = resources
+        self.env = env
+
+        super().__init__(asynchronous=asynchronous, **kwargs)
+        if not self.asynchronous:
+            self._loop_runner.start()
+            self.sync(self._start)
+
+    async def _start(self):
+        self.core_api = kubernetes.client.CoreV1Api()
+        self.custom_api = kubernetes.client.CustomObjectsApi()
+        data = build_cluster_spec(
+            self.name, self.image, self.replicas, self.resources, self.env
+        )
+        cluster = self.custom_api.create_namespaced_custom_object(
+            group="kubernetes.dask.org",
+            version="v1",
+            plural="daskclusters",
+            namespace=self.namespace,
+            body=data,
+        )
```

Comment:
The `NewConnectionError` is coming from here. I'm not sure why I can't create custom resources here, but I can [here](https://github.com/dask/dask-kubernetes/pull/392/files#diff-d63febde65b9397396016375ad5bee42b517b65a423e69918aebac8c6c0b07a9R187-R193).

Reply:
I wonder if it can't connect to the Kubernetes API. Something to do with kubeconfig, maybe?

Reply:
This might be important. I don't see it here.

https://github.com/dask/dask-kubernetes/blob/2aa7212cc7fbffd25caa8cf474d950dd876821d0/dask_kubernetes/core.py#L561